src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
```

Comment:
You can simplify this condition since the case where 'days' is less than 3 is already handled by the previous 'if' statement. Therefore, you only need to check if 'days' is less than the long-term threshold.